### ISSUE TITLE
Incorrect snake_case error

### ISSUE THREAD

**Issue by sazzer** [yeya24/promlinter#49]
Hi there,

I've just started using promlinter via golangci, and I'm getting a lint error that I don't believe is correct:
> application/infrastructure/httpserver/metrics.go:19:38: Metric: http_requests_total Error: label names should be written in 'snake_case' not 'camelCase' (promlinter)

The error is coming from this code:
```
	counter := prometheus.NewCounterVec(prometheus.CounterOpts{
		Namespace: "http",
		Name:      "requests_total",
		Help:      "Count of the number of HTTP requests processed",
	}, []string{"method", "uri", "statusCode", "error"})
```

Except that the metric name is clearly already in snake case - especially since that's included in the error message itself!

The actual generated metrics are as follows too:
```
# HELP http_requests_total Count of the number of HTTP requests processed
# TYPE http_requests_total counter
http_requests_total{error="false",method="GET",statusCode="200",uri="/debug/metrics"} 1
```

So again, the name is clearly already in snake case.

Am I missing something here?

Cheers

**Comment by yeya24** [comment]
Hey thanks for the issue. The error message is not clear. I agree it should at least point that labels that have the issue.

It is not the metric name that failed the lint. It is the label statusCode which uses camel case that failed the lint.

**Comment by sazzer** [comment]
Aha - so it is. That's removed the `nolint` from my code. Thanks :)

It even actually says "label names" in the error! I'm just new to Go Prometheus and had taken that to mean the name of the metric itself - when I've done this in Java before we called them "tags" instead of "labels" which probably caused the confusion!

Cheers
